docs/metrics-analysis-prometheus/prometheus.md

```diff
@@ -36,20 +38,17 @@ within a specified time range.
    sum of the specified metric. In our example, it calculates the
    cumulative sum of the github_actions_job_cost_count_total metric,
    representing the total job cost count.
-3. Within the curly braces {}, we apply filters to narrow down the
-   data based on specific criteria. The `$variable` refers to the filter
-   variables that you can specify at the top of the page.
-4. The `[$__range]` part specifies the time range for the query.
-   It uses the `$__range` variable, which represents the selected time
-   range in Grafana.
-5. The `by (repository)` clause groups the data by the repository field.
+3. The `[5m]` part specifies the time range for the query.
+4. The `by (repository)` clause groups the data by the repository field.
    This enables the query to calculate the cost sum for each repository individually.
-6. The expression `> 0` filters the query results to only include
+5. The expression `> 0` filters the query results to only include
    repositories with a value greater than zero.
 
-It's also possible to combine different queries in Grafana. For example, one
-query dividing by another.
-
-Lastly, it is also important to understand `__interval`. `__interval`
-represents the time interval between data points, whereas `__range`
-represents the selected time range for the query.
+!!! info
+   You can also use Grafana, it enhances the visualization of Prometheus data and
```

Comment:
Same trunk story as explained above, leaving suggestion to not forget the change
```suggestion

   You can also use Grafana, it enhances the visualization of Prometheus data and
```